src/virtualenv/create/via_global_ref/builtin/pypy/pypy3.py

```diff
@@ -37,15 +35,6 @@ def _shared_libs(cls):
     def to_lib(self, src):
         return self.dest / "lib" / src.name
 
-    @classmethod
-    def sources(cls, interpreter):
-        for src in super(PyPy3Posix, cls).sources(interpreter):
-            yield src
-        host_lib = Path(interpreter.system_prefix) / "lib"
-        if host_lib.exists() and host_lib.is_dir():
-            for path in host_lib.iterdir():
-                yield PathRefToDest(path, dest=cls.to_lib)
```

Comment:
I'm fairly sure there was a pypy3 variant that worked like this (over a year ago when I wrote this). Doesn't this drop support for those?

Reply:
Ok, let's err on the side of caution and stick with @mgorny 's fix

Reply:
I'd actually prefer if we'd not do this :blush: but make it version-specific; if you can dig out from pypy when this change happened?

Reply:
Before PyPy's reorganization of the directory layout for python3.8 there were no directories in `prefix/lib`. 

It seems this code was added in #1503 in a blanket change to "support c-extensions inside virtual environments". The issue pointed to is #1448, but that issue only mentions include files, not lib files. It seems to relate to issue #1023, but that one is windows only and deals with the libs directory, not the lib one.

Reply:
Hmm, in that case I'm happy to go with your original proposal, sorry for the back and forth.

Reply:
cool, updating